units/src/parse.rs

```diff
@@ -263,19 +264,20 @@ pub fn hex_u128_prefixed(s: &str) -> Result<u128, PrefixedHexError> {
 ///
 /// # Errors
 ///
-/// - If input string contains a `0x` (or `0X`) prefix.
-/// - If input string is not a valid hex encoding of a `u128`.
+/// - If the input string contains a `0x` (or `0X`) prefix.
+/// - If the input string is not a valid hex encoding of a `u128`.
 pub fn hex_u128_unprefixed(s: &str) -> Result<u128, UnprefixedHexError> {
     let checked = hex_check_unprefixed(s)?;
     Ok(hex_u128_unchecked(checked)?)
 }
 
-/// Parses a `u128` from a hex string.
+/// Parses a `u128` an unprefixed hex string without first checking for a prefix.
 ///
 /// # Errors
 ///
-/// - If input string is not a valid hex encoding of a `u128`.
-/// - With `InvalidDigit` due to the `x` if there is a prefix.
+/// - If the input string contains a `0x` (or `0X`) prefix an `InvalidDigit` error is returned due
```

Comment:
Note, will need changing if you choose to change the docs on `hex_u32_unchecked`.